winrt/test.managed/PixelShaderEffectTests.cs

```diff
@@ -1216,13 +1216,13 @@ public void PixelShaderEffect_InputRectTooBigError()
                 // Drawing with an infinite sized input is not!
                 effect.Source1 = new ColorSourceEffect();
 
-                Utils.AssertThrowsException<Exception>(() =>
+                Assert.ThrowsException<Exception>(() =>
                 {
                     using (var ds = renderTarget.CreateDrawingSession())
                     {
                         ds.DrawImage(effect);
                     }
-                }, " graph could not be rendered with the context's current tiling settings. (Exception from HRESULT: 0x88990027)");
+                });
```

Comment:
I assume this is also a fix for non-en culture?   Can the test still look for presence of 0x88990027 in the exception string?  Without that, it's not really testing for the specific failure condition that is expected any more.

Reply:
Yes, on my box the message was very different. I was thinking the same. Will add a check for HResult